<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Auto save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbAutoSaveInterval">
     <property name="toolTip">
      <string>Interval between two automatic saves.</string>
     </property>
     <property name="suffix">
      <string> min</string>
     </property>
     <property name="minimum">
      <number>6</number>
     </property>
     <property name="maximum">
      <number>2880</number>
     </property>
     <property name="singleStep">
      <number>6</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Data limit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbDataLengthLimit">
     <property name="toolTip">
      <string>Number of data points to keep at least in memory, if the data limit option is activated.</string>
     </property>
     <property name="suffix">
      <string> samples</string>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Save file path</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="wgSavePath" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="leSavePath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbSavePath">
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="gbPlotDefaults">
     <property name="title">
      <string>Plot defaults</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>AutoCut</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbAutoCut">
        <property name="toolTip">
         <string>Default number of samples to show for new plots.</string>
        </property>
        <property name="suffix">
         <string> samples</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Grid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cbGrid">
        <property name="toolTip">
         <string>Start the plots with a grid.</string>
        </property>
        <property name="text">
         <string>Start with grid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
